R/toml_serialization.R

```diff
@@ -19,7 +19,7 @@
 #'     one line of the resulting output.
 #' @examples
 #' # Produces [workspace] with no children
-#' to_toml(workspace = )
+#' to_toml(workspace = NULL)
```

Comment:
Adding explicit `NULL` in the documentation. Using an empty argument just feels.... wrong.

Reply:
Yes, if we're sure that `NULL` is equivalent (and I think it is here), then I prefer this too.

Reply:
```r
identical(to_toml(workspace = ), to_toml(workspace = NULL))
#> TRUE
```